[IMP] POS: remove vantiv integration 15.0

dikd-odoo opened the pull request:
[CANCELLED] Moved to 17.5 

Removed Vantiv integration from 15.0 Sales POS documentation:

    Deleted Vantiv page from Point of Sale > Payment Methods > Terminals
    Updated Payment Terminals page content and image

Previously peer reviewed by JERO - PR #10426 

[Task](https://www.odoo.com/odoo/my-tasks/4077781?cids=3)

forward port up to 16.0

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/10429.png)](https://mergebot.odoo.com/odoo/documentation/pull/10429)

dikd-odoo commented:
@odoo/us-doc-review @StraubCreative Hello! This PR is ready for final review. The changes were previously peer reviewed and approved by JERO on PR #10426. Moved to this PR as it is a change starting in 15.0 onward.

StraubCreative commented on content/applications/sales/point_of_sale/payment_methods/terminals.rst:
```diff
@@ -15,15 +15,14 @@ Configuration
 :guilabel:`Payment Terminals` section, and tick your terminal's checkbox.
 
-.. image:: terminals/settings-pt.png
+.. image:: terminals/payment-terminals.png
```
Since this screenshot was taken outside of Odoo 15, I'd remove it.
In both Odoo 15 and Odoo 16, Vantiv still appears as an option...

Can add it back in on the 17.0 forward-port PR, if you want!

StraubCreative commented on content/applications/sales/point_of_sale/payment_methods/terminals/vantiv.rst:
```diff
@@ -1,45 +0,0 @@
-======
-Vantiv
```
Need to add a 301 redirect for the deleted file, to the `redirects.txt` file (can point to `terminals.rst`)

StraubCreative reviewed (approved):
Hi @dikd-odoo 

Approving with comments. 
See below, address the two items, and can move to tech review with @samueljlieber after, ty!

dikd-odoo commented:
Thank you @StraubCreative! Implemented your changes here (15.0 PR) with the note that Vantiv is still currently available in 15.0 and 16.0. (Noted below to SALI)

Hi @samueljlieber, this PR and its sibling #10426 are ready for technical review! Some notes:

We were assigned [a task](https://www.odoo.com/odoo/my-tasks/4077781) to remove Vantiv from our documentation as it is being deprecated from PoS terminals in 17.5. Because Vantiv is still currently available in 15.0-16.0, this PR is removing the vantiv.rst file and images but without the updated screenshot of available PoS terminals. 

The sibling PR #10426 is the update for 17.0+onward. I removed the vantiv.rst file and images and added an updated screenshot of available PoS terminals. 

Please let me know if you have any questions. Thank you!
